.travis.yml

```diff
@@ -55,11 +55,10 @@ matrix:
       before_install:
         - docker -v
         - docker-compose -v
-        - wget -qO- 'https://github.com/tianon/pgp-happy-eyeballs/raw/master/hack-my-builds.sh' | bash
```

Comment:
Is there no other way than to pipe a shell script to bash? Maybe by using a local copy?